addons/core/translations/en-us.yaml

```diff
@@ -133,3 +133,28 @@ settings:
   alerts:
     cache-daemon: There may be a problem with the cache daemon
     client-agent: There may be a problem with the client agent
+worker-filter-generator:
+  toggle:
+    title: Show filter generator
+    description: The generator will automatically generate a pre-formatted filter for you. You can copy the result to the editor above.
+  filter-generator:
+    title: Filter generator
+    description: Choose what you want to format into a filter.
+    link: Learn more about formatting a worker filter.
+    input-values:
+      title: Input values
+      description: Next, input what you want to format.
+    formatted-result:
+      title: Formatted result
+      description: Copy and paste this into the editor above.
+      label: Generated result
+    tag:
+      label: Tag
+      helper: Format a worker tag
+    name:
+      label: Name
+      helper: Format a worker name
+    operator:
+      matches: matches
+      contains: contains
+      default: --Select an option--
```

Comment:
We also have a very similar copy here https://github.com/hashicorp/boundary-ui/blob/b0271096ccc38c3bd3d09ed7efe72339cde689a2/addons/core/translations/en-us.yaml#L39

Do we need more than one?

Reply:
Good catch. I will update that to the existing one so it is the same across the app.

Reply:
Fixed.